project/project.sbt

```diff
@@ -2,7 +2,7 @@ import java.util.Locale
 
 libraryDependencies += "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
 
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.6")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
I'm not 100% sure here, but I think you will need sbt-pgp version `1.1.0-M1` here.